packages/truffle-core/test/config.js

```diff
@@ -6,7 +6,7 @@ var Contracts = require("truffle-workflow-compile");
 var Ganache = require("ganache-core");
 var provision = require("truffle-provisioner");
 var Resolver = require("truffle-resolver");
-var Artifactor = require("truffle-artifactor");
+const { Artifactor } = require("truffle-artifactor");
```

Comment:
The more I see this, the more I think we should just keep our `index.js` to maintain the default export, where all that file would do is import from `./dist`

Reply:
Is there a way to `export default` w/o breaking things? Importing destructured like this seemed prettier than having to do `const Artifactor = require("truffle-artifactor").default` everywhere.

Reply:
You have to do it in JS I believe

Reply:
Or maybe you can finagle the tsconfig to make it work the way you want.